### ISSUE TITLE
php 5.6 on Centos 7.1.1503

### ISSUE THREAD

**Issue by ajasak** [mayflower/puppet-php#131]
I'm trying to install php 5.6 on Centos 7.1.1503 by adding manage_repos => true.
It ends with error:
```
Error: Execution of '/usr/bin/yum -d 0 -e 0 -y install php-common' returned 1: Error: Package: php-pecl-zip-1.13.1-1.el7.remi.5.6.x86_64 (remi-php56)
           Requires: libzip.so.4()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Error: /Stage[main]/Php::Packages/Package[php-common]/ensure: change from absent to present failed: Execution of '/usr/bin/yum -d 0 -e 0 -y install php-common' returned 1: Error: Package: php-pecl-zip-1.13.1-1.el7.remi.5.6.x86_64 (remi-php56)
           Requires: libzip.so.4()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Error: Execution of '/usr/bin/yum -d 0 -e 0 -y install php-cli' returned 1: Error: Package: php-pecl-zip-1.13.1-1.el7.remi.5.6.x86_64 (remi-php56)
           Requires: libzip.so.4()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
Error: /Stage[main]/Php::Packages/Package[php-cli]/ensure: change from absent to present failed: Execution of '/usr/bin/yum -d 0 -e 0 -y install php-cli' returned 1: Error: Package: php-pecl-zip-1.13.1-1.el7.remi.5.6.x86_64 (remi-php56)
           Requires: libzip.so.4()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```
Solution for this error is to add remi or remi-safe repository to manifests/repo/redhat.pp:
```
contain "::yum::repo::remi"
```
or
```
yum::managed_yumrepo { 'remi-safe':
    descr         => 'Les RPM de remi pour Enterpise Linux $releasever - $basearch - PHP 5.6',
    mirrorlist    => 'http://rpms.famillecollet.com/enterprise/$releasever/safe/mirror',
    enabled       => 1,
    gpgcheck      => 1,
    gpgkey        => 'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-remi',
    gpgkey_source => 'puppet:///modules/yum/rpm-gpg/RPM-GPG-KEY-remi',
    priority      => 1,
  }
```
Can you add one of these solutions to module?

**Comment by fpletz** [comment]
After reading http://blog.remirepo.net/pages/English-FAQ#safe as a non-RedHat user, I would rather go with `remi-safe`. Given that `remi` is already available in the `yum` module, we should for the sake of compatibility go with including `::yum::repo::remi` to prevent users from adding the `remi-safe` repo multiple times or run into duplicated resources errors.

Will do before 4.0.0. Should be released later this week.

**Comment by edestecd** [comment]
I am having this issue as well.  I vote for:
```puppet
contain '::yum::repo::remi'
```

Shall I submit a pull request or is someone else doing it?

**Comment by fpletz** [comment]
@edestecd I wanted to do it, but haven't gotten around lately. Please do! :)

**Comment by heavyengineer** [comment]
I submitted a PR just now. Works for me. https://github.com/mayflower/puppet-php/pull/148

**Comment by edestecd** [comment]
Awesome!

**Comment by fpletz** [comment]
Fixed by #148.
